configapi/routers.go

```diff
@@ -144,4 +144,40 @@ var routes = Routes{
 		"/network-slice/:slice-name",
 		NetworkSliceSliceNamePut,
 	},
+	{
+		"GetGnbs",
+		http.MethodGet,
+		"/inventory/gnb",
+		GetGnbs,
+	},
+	{
+		"PostGnb",
+		http.MethodPost,
+		"/inventory/gnb/:gnb-name",
+		PostGnb,
+	},
+	{
+		"DeleteGnb",
```

Comment:
IT seems good to have these APIs but does backend support it? e.g. if gNB is deleted then does the backend support handling? I am sure it may work at webconsole level but at NF level handling

Reply:
What do we achieve my doing just front end ?